qprofiler2/test/org/qcmg/qprofiler2/summarise/KmersSummaryTest.java

```diff
@@ -214,7 +210,7 @@ public void toXmlTest() throws IOException, DOMException, ParserConfigurationExc
 	
 	
 	@Test
-	public void bothForwardTest() throws IOException {
+	public void bothForwardTest() throws Exception {
```

Comment:
are any exceptions thrown from this method? If not, please remove, otherwise be more specific in terms of the type of exception